Tasks/AzureRmWebAppDeployment/LegacyUtils/Utility-Legacy.ps1

```diff
@@ -185,4 +190,30 @@ function Run-MsDeployCommand
     Write-Host (Get-LocalizedString -Key "Running msdeploy command: `n`t{0}" -ArgumentList $msDeployCmdForLogs)
     Run-Command -command $msDeployCmd
     Write-Host (Get-LocalizedString -Key "msdeploy command ran successfully.")
+}
+
+function Get-UserAgentString
```

Comment:
Is this function going to be reused across multiple Azure tasks? If so, should it reside in common?